ext/project_data.yaml

```diff
@@ -36,7 +36,7 @@ gem_rdoc_options:
 gem_platform_dependencies:
   universal-darwin:
     gem_runtime_dependencies:
-      CFPropertyList: '~> 2.2'
+      CFPropertyList: '>= 3.0.6'
```

Comment:
maybe `~> 3.0` so we don't accept 4?

Reply:
I opted for `'>= 3.0.6', '< 4'` so we always get the Ruby >= 3.2 compatible version, but don't bump up to the next major version.